tendrl/monitoring_integration/sync/dashbaord_sync.py

```diff
@@ -58,6 +58,7 @@ def refresh_dashboard(self):
                 AttributeError,
                 req_excep.ConnectionError,
                 TypeError,
+                ValueError,
```

Comment:
json.loads will throw this error, we are using JSON.loads in few places so we need to handle that exception

Reply:
ok